src/mod/applications/mod_avmd/mod_avmd.c

```diff
@@ -1319,6 +1335,10 @@ static switch_status_t avmd_parse_cmd_data_one_entry(char *candidate, struct avm
         if(avmd_parse_u8_user_input(val, &settings->detectors_lagged_n, 0, UINT8_MAX) == -1) {
             return SWITCH_STATUS_FALSE;
         }
+    } else if (!strcmp(key, "detectors_idle_time")) {
+        if(avmd_parse_u16_user_input(val, &settings->detectors_idle_time, 0, UINT16_MAX) == -1) {
```

Comment:
perhaps to add a space after `if`? just to be consistent

Reply:
I'm also following the module author on this function. If you see above this changes it also have the same pattern. Adding a space seems a need to readjust all the if conditions above.